src/sentry/ingest/consumer/attachment_event.py

```diff
@@ -36,7 +39,7 @@ def decode_and_process_chunks(
     metrics.distribution(
         "ingest_consumer.payload_size",
         len(raw_payload),
-        tags={"consumer": consumer_type},
+        tags={"consumer": str(consumer_type)},
```

Comment:
```suggestion
        tags={"consumer": consumer_type.value},
```
...to match the previous values